File: app/src/main/java/com/moviepass/activities/TicketType.java
package com.moviepass.activities;

/**
 * Created by o_vicarra on 11/9/17.
 */

import org.parceler.Parcel;

@Parcel

public class TicketType {


    String showtimeId;
    String id;
    int quantity;
    int price;

    public TicketType() {
    }

    public String getId() {
        return id;
    }

}

File: app/src/main/java/com/mobile/reservation/TicketType.kt
package com.mobile.reservation

import android.os.Parcelable

import kotlinx.android.parcel.Parcelize

@Parcelize
class TicketType(var showtimeId: String? = null,
                 var id: String? = null,
                 var quantity: Int = 0,
                 var price: Int = 0
) : Parcelable
